src/main/java/com/epam/test_generator/entities/Status.java

```diff
@@ -9,11 +9,11 @@
  */
 public enum Status {
 
-    NOT_DONE("Not done"),
-    NOT_RUN("Not run"),
-    PASSED("Passed"),
-    FAILED("Failed"),
-    SKIPPED("Skipped");
+    NOT_DONE("NOT DONE"),
```

Comment:
I doubt if we need separate getters *getDisplayedName* when *getStatus* returns exact same result.

Reply:
Not quite, getStatus method returns enum as string (left value) and getDisplayedName returns the string that belongs to enum (right value).